client/src/core/client/ui/components/icons/ActiveNotificationBellIcon.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FunctionComponent } from "react";
+
+const ActiveNotificationBellIcon: FunctionComponent = () => {
+  return (
+    <svg viewBox="-1.25 0 21 21" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <path
+        d="M13.2291 4.85596C13.2291 5.519 13.4925 6.15488 13.9614 6.62372C14.4302 7.09257 15.0661 7.35596 15.7291 7.35596C16.3922 7.35596 17.0281 7.09257 17.4969 6.62372C17.9657 6.15488 18.2291 5.519 18.2291 4.85596C18.2291 4.19292 17.9657 3.55703 17.4969 3.08819C17.0281 2.61935 16.3922 2.35596 15.7291 2.35596C15.0661 2.35596 14.4302 2.61935 13.9614 3.08819C13.4925 3.55703 13.2291 4.19292 13.2291 4.85596Z"
+        fill="#EB5757"
```

Comment:
Can you please remove? I don't think we want to set a fill color on the icon, but rather add it with css so that it's more reusable.

Reply:
this fill colour is specifically for the red dot on the bell, it's not a shared fill colour, I'll convert it to a configurable value via CSS. Thanks for catching this!